sdk/storage/storage-file/samples/README.md

```diff
@@ -0,0 +1,45 @@
+# Getting started with samples
+
+## Install the library
+
+Run the below in your samples folder to install the npm package for Azure Storage File library.
+
+```bash
+npm install @azure/storage-file
+```
+
+## Get account credentials of your storage account
+
+- In the [Azure Portal](https://portal.azure.com), go to **Dashboard > Storage > _your-storage-account_**.
+- Note down the "AccountName", "AccountKey" obtained at **Access keys** and "AccountSAS" from **Shared access signature** under **Settings** tab.
+  Before running any of the samples, update with the credentials you have noted down above.
+
+## Running a Javascript sample
+
+Copy the sample to your samples folder and use `node` to run it.
+
+```bash
+node sample.js
+```
+
+## Running a Typescript sample
+
+If you don't have Typescript installed, then use `npm` to install it first.
+
+```bash
+npm install -g typescript
+```
+
+One way to run Typescript samples is to use `ts-node`. To install `ts-node`, run the below in your sample folder
+
+```bash
+npm install ts-node
```

Comment:
Should `ts-node` be installed globally as well in order to run it like below?

Reply:
Yes, we need to install `ts-node` globally in order to run the command.
Updated!